Actions/RunPipeline/RunPipeline.ps1

```diff
@@ -507,6 +508,22 @@ try {
         -appBuild $appBuild -appRevision $appRevision `
         -uninstallRemovedApps
 
+    # If any apps were downloaded as part of incremental builds, we should remove them again after the build
```

Comment:
The logic is fine, but this should only happen if this is a PR build - if this is a normal build we will publish downloaded apps and downloaded projects on the build.
You could check ENV:GITHUB_EVENT_NAME -like 'pull_request*' before running this section.